src/layouts/case-study.hbs

```diff
@@ -22,6 +22,16 @@
         {{#with @root.page.title}}
           <h1 class="page">{{{this}}}</h1>
         {{/with}}
+        {{#with (get-page-cards @root.page 'manual' true)}}
```

Comment:
you will need to create an helper function because the `book-cover` function expect the following format:

```
{
  "title": "",
  "color": ""
  "id: "",
  "url": ""
}
```

Reply:
technically we could create a generic helper function that retrieves the color and title (and id?) from an AsciiDoc attribute